### DISCUSSION TITLE
How to get cookie from HttpRequest?

### DISCUSSION THREAD

**Discussion by shivshankardayal** [ntex-rs/ntex#218] [Q&A] [answered]
I see that there is `head, path, payload, app_state, rmap` and `pool` in `HttpRequest` but no cookie. How can I get cookies submitted in the request?

**Comment by shivshankardayal** [comment]
I see that I should be able to get it from a header named "Cookie".

**Comment by fafhrd91** [comment, marked as answer]
there is [cookies](https://docs.rs/ntex/0.7.2/ntex/http/trait.HttpMessage.html#method.cookies) method. you have to enable “cookie” feature

**Comment by shivshankardayal** [comment]
Muck thanks Nikolay!!!
